test/config/utility.cc

```diff
@@ -41,6 +41,17 @@ const std::string ConfigHelper::BASE_CONFIG = R"EOF(
         port_value: 0
 )EOF";
 
+const std::string ConfigHelper::REDIS_PROXY_CONFIG = BASE_CONFIG + R"EOF(
```

Comment:
Since this is an extension, let's keep this inside the redis integration test file.

Reply:
done